osquery/tables/system/darwin/apps.cpp

```diff
@@ -47,6 +51,109 @@ const std::vector<std::string> kHomeDirSearchPaths = {
     "Applications", "Desktop", "Downloads",
 };
 
+enum AppSchemeFlags {
+  kSchemeNormal = 0,
+  // Default flag from the list of schemes on a default OS X 10.10 install.
+  kSchemeSystemDefault = 1,
+  // Protected flag from Apple Reference: Inter-app Communication
+  kSchemeProtected = 2,
+};
+
+const std::map<std::string, unsigned short> kApplicationSchemes = {
```

Comment:
if this was in some sort of function, we could make the API a bit more future proof for when this list is dynamically generated?

```cpp
const std::map<std::string, unsigned short> getApplicationSchemes() {
  return {
    {"account", 0},
    // ...
  };
}
``